examples/convert_little_endian_to_big.cpp

```diff
@@ -30,10 +30,10 @@ int main()
     std::cout << "in is: " << std::hex << in << std::endl;
     // If the host is little endian, the put function
     // should change the byte order (no change for big endian)
-    endian::convert_endian::put32(in, data);
+    endian::little_endian::put32(in, data);
```

Comment:
Let's also rename the variable `in` to `input`. I did not understand why it was called `in`. Try to remember that we have a no-abbreviation policy ;)